queries/groovy/highlights.scm

```diff
@@ -1,70 +1,261 @@
-(unit
-  (identifier) @variable)
+[
+  "!in"
+  "!instanceof"
+  "as"
+  "assert"
+  "case"
+  "catch"
+  "class"
+  "def"
+  "default"
+  "else"
+  "extends"
+  "finally"
+  "for"
+  "if"
+  "import"
+  "in"
+  "instanceof"
+  "package"
+  "pipeline"
+  "return"
+  "switch"
+  "try"
+  "while"
+  (break)
+  (continue)
+] @keyword
 
-(string
-  (identifier) @variable)
+[
+  "true"
+  "false"
+] @boolean
 
-(escape_sequence) @string.escape
+(null) @constant
 
-(block
-  (unit
-    (identifier) @module))
+"this" @variable.builtin
 
-(func
-  (identifier) @function)
+[
+  "int"
+  "char"
+  "short"
+  "long"
+  "boolean"
+  "float"
+  "double"
+  "void"
+] @type.builtin
 
-(number) @number
+[
+  "final"
+  "private"
+  "protected"
+  "public"
+  "static"
+  "synchronized"
+] @type.qualifier
 
-((identifier) @boolean
-  (#any-of? @boolean "true" "false" "True" "False"))
+(comment) @comment
 
-((identifier) @constant
-  (#lua-match? @constant "^[A-Z][A-Z%d_]*$"))
+(shebang) @comment
 
-((identifier) @constant.builtin
-  (#eq? @constant.builtin "null"))
+(string) @string
 
-((identifier) @type
-  (#any-of? @type "String" "Map" "Object" "Boolean" "Integer" "List"))
+(string
+  (escape_sequence) @operator)
 
-((identifier) @function.builtin
-  (#any-of? @function.builtin "void" "id" "version" "apply" "implementation" "testImplementation" "androidTestImplementation" "debugImplementation"))
+(string
+  (interpolation
+    "$" @operator))
 
-((identifier) @keyword
-  (#any-of? @keyword "static" "class" "def" "import" "package" "assert" "extends" "implements" "instanceof" "interface" "new"))
+"(" @punctuation.bracket
 
-((identifier) @type.qualifier
-  (#any-of? @type.qualifier "abstract" "protected" "private" "public"))
+")" @punctuation.bracket
 
-((identifier) @keyword.exception
-  (#any-of? @keyword.exception "throw" "finally" "try" "catch"))
+"[" @punctuation.bracket
```

Comment:
Group all of these `@punctuation.bracket` into one

e.g.
```
[
  ...
]
```

Reply:
Same for `@punctuation.delimiter`.